src/engine-actions/cancel-swap.js

```diff
@@ -3,8 +3,8 @@ const { cancelInvoice } = require('../lnd-actions')
 /**
  * Cancels the invoice for a swap
  *
- * @param {Bytes} swapHash - hash of the swap
- * @returns {Promise}
+ * @param {string} swapHash - hash of the swap
+ * @returns {Promise<Object>}
```

Comment:
can `Object` be more specific here?

Reply:
Yeah, but I'll leave refinements for later; there are tons of things that would be nice to refine.